scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -487,3 +483,27 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
     except exp.ExportProjectError as err:
         print(err)
         sys.exit(1)
+
+# This code is only run by the 'export-project' O3DE CLI command
+
+if "o3de_context" in globals():
+    global o3de_context
+    global o3de_logger
+
+    # Resolve the export config
+    try:
+        if o3de_context is None:
+            project_name, project_path = command_utils.resolve_project_name_and_path()
+            export_config = exp.get_export_project_config(project_path=project_path)
+        else:
+            export_config = exp.get_export_project_config(project_path=o3de_context.project_path)
+    except command_utils.O3DEConfigError:
+        o3de_logger.debug(f"No project detected at {os.getcwd()}, getting default settings from global config instead.")
+        project_name = None
+        export_config = exp.get_export_project_config(project_path=None)
+
+    args = export_standalone_parse_args(o3de_context, export_config)
+
+    export_standalone_run_command(o3de_context, args, export_config, o3de_logger)
+    o3de_logger.info("Finished exporting project")
```

Comment:
Good idea for this info, but I would also add helpful information like:
`Finished exporting project to ...` where ... is the folder where the projects were exported to.